template/makeBp.js

```diff
@@ -286,16 +286,16 @@ exports.generateApiSummary = generateApiSummary;
 
 function getAuthBools(tabbing, apiCall) {
     var output = "";
-    if (apiCall.auth === "EntityToken" || apiCall.url === "/Authentication/GetEntityToken" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
+    if (apiCall.auth === "EntityToken" || apiCall.url === "/Authentication/GetEntityToken")
         output += tabbing + "manager->useEntityToken = true;\n";
-    if (apiCall.auth === "SecretKey" || apiCall.url === "/Authentication/GetEntityToken" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
+    if (apiCall.auth === "SecretKey" || apiCall.url === "/Authentication/GetEntityToken")
         output += tabbing + "manager->useSecretKey = true;\n";
-    if (apiCall.auth === "SessionTicket" || apiCall.url === "/Authentication/GetEntityToken" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
+    if (apiCall.auth === "SessionTicket" || apiCall.url === "/Authentication/GetEntityToken")
         output += tabbing + "manager->useSessionTicket = true;\n";
 
-    if (apiCall.result === "LoginResult" || apiCall.result === "RegisterPlayFabUserResult" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
+    if (apiCall.result === "LoginResult" || apiCall.result === "RegisterPlayFabUserResult")
         output += tabbing + "manager->returnsSessionTicket = true;\n";
-    if (apiCall.url === "/Authentication/GetEntityToken" || apiCall.url === "/GameServerIdentity/AuthenticateGameServerWithCustomId")
```

Comment:
I think you wanted to keep this line, as it declares the method as returning an entity token.